exec/cnex/array.c

```diff
@@ -40,24 +40,58 @@ Array *array_createArrayFromSize(size_t iElements)
 
 Array *array_expandArray(Array *self, size_t iElements)
```

Comment:
What's the difference between calling array_expandArray and calling array_resizeArray? Why not just have array_resizeArray which can make the array both bigger and smaller?

Reply:
Good call.  Removed expandArray(), in favor of resizeArray().